samples/apps/copilot-chat-app/webapp/src/libs/services/ContentModerationService.ts

```diff
@@ -0,0 +1,29 @@
+import { BaseService } from './BaseService';
+
+type IContentModerationResult = Record<string, { category: string; riskLevel: number }>;
+
+export class ContentModerationService extends BaseService {
+    public analyzeImageAsync = async (base64Image: string, accessToken: string): Promise<IContentModerationResult> => {
```

Comment:
nit: return type isn't needed since its inferred

Reply:
It doesn't matter, leaving as is